jwst/regtest/test_nirspec_ifu.py

```diff
@@ -47,6 +47,7 @@ def run_spec2(jail, rtdata_module):
     return rtdata
 
 
+@pytest.mark.skip(reason="single exposure processing is redundant with calspec2")
```

Comment:
This is the fixture. You actually want to `pytest.mark.skip` the `test_spec3` test below that uses this `run_spec3` fixture.

Reply:
Thanks. Fixing that.